modules/passkey/test/SafeWebAuthnSignerProxyFactory.spec.ts

```diff
@@ -9,16 +9,18 @@ import { encodeWebAuthnSigningMessage } from './utils/webauthnShim'
 describe('SafeWebAuthnSignerFactory', () => {
   const setupTests = deployments.createFixture(async () => {
     const { SafeWebAuthnSignerFactory } = await deployments.fixture()
+
     const factory = await ethers.getContractAt('SafeWebAuthnSignerFactory', SafeWebAuthnSignerFactory.address)
 
+    const singletonAddress = await factory.SINGLETON()
```

Comment:
This is only needed for a test, I wouldn't consider it as being part of the "Setup".